addons/website/views/snippets/snippets.xml

```diff
@@ -139,7 +139,7 @@
                     </t>
                     <t t-snippet="website.s_card" t-thumbnail="/website/static/src/img/snippets_thumbs/s_card.svg"/>
                     <t t-snippet="website.s_share" t-thumbnail="/website/static/src/img/snippets_thumbs/s_share.svg"/>
-                    <t t-snippet="website.s_social_media" t-thumbnail="/website/static/src/img/snippets_thumbs/s_facebook_page.svg"/> <!-- TODO replace by new thumbnail-->
+                    <t t-snippet="website.s_social_media" t-thumbnail="/website/static/src/img/snippets_thumbs/s_social_media.svg"/>
```

Comment:
Since this work is mostly about the SVG, change the authorship of the commit to the person who did it. You can add yourself as co-author thanks to the `Co-authored-by:` instruction :wink: